### PR TITLE
[MIG][15.0] connector + component_event + test_connector

### PR DESCRIPTION
**Pull request by i-vyshnevska**

### CODE REVIEW

**Comment by simahawk** [comment]
`test_connector` should be migrated too

File: component_event/tests/test_event.py [hunk 1]
```diff
@@ -47,6 +47,11 @@ def doClassCleanups(cls):
                 except Exception:
                     cls.tearDown_exceptions.append(sys.exc_info())
```
Comment (simahawk) [inline component_event/tests/test_event.py]: the block above should be removed

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by i-vyshnevska** [comment]
@simahawk ready for review

File: connector/components/synchronizer.py [hunk 2]
```diff
@@ -55,7 +56,7 @@ def run(self):
     def mapper(self):
         """Return an instance of ``Mapper`` for the synchronization.
 
-        The instanciation is delayed because some synchronisations do
+        The instantiations is delayed because some synchronizations do
```
Comment (simahawk) [inline connector/components/synchronizer.py]:
```suggestion
        The instantiation is delayed because some synchronizations do
```

Comment (yankinmax) [inline connector/components/synchronizer.py]: as you're fixing mistakes, you can also choose between `instantiations are` and `instantiation is`

File: component_event/tests/test_event.py [hunk 3]
```diff
@@ -162,7 +139,7 @@ class TestEvent(ComponentRegistryCase):
 
     def setUp(self):
         super(TestEvent, self).setUp()
```
Comment (yankinmax) [inline component_event/tests/test_event.py:141]:
```suggestion
        super().setUp()
```

File: component_event/tests/test_event.py [hunk 4]
```diff
@@ -392,6 +369,7 @@ class TestEventFromModel(TransactionComponentRegistryCase):
 
     def setUp(self):
         super(TestEventFromModel, self).setUp()
```
Comment (yankinmax) [inline component_event/tests/test_event.py:371]:
```suggestion
        super().setUp()
```

File: component_event/tests/test_event.py [hunk 5]
```diff
@@ -21,31 +20,9 @@
...
-            cls.tearDown_exceptions = []
-            while cls._class_cleanups:
-                function, args, kwargs = cls._class_cleanups.pop()
-                try:
-                    function(*args, **kwargs)
-                except Exception:
-                    cls.tearDown_exceptions.append(sys.exc_info())
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.test_sequence = 0
```
Comment (yankinmax) [inline component_event/tests/test_event.py:25]: I didn't meet such code before. Could you pls explain what `self.test_sequence = 0` is for?

**Comment by saxomanu** [comment]
Is it ready to merge ? what is missing ? Can i help ?

**Comment by i-vyshnevska** [comment]
@simahawk can you pass another review

File: component_event/tests/test_event.py (hunk 5 above)
Comment (simahawk) [reply component_event/tests/test_event.py:25]: @i-vyshnevska ?

**Comment by simahawk** [comment]
@i-vyshnevska pls squash when done

File: component_event/tests/test_event.py (hunk 5 above)
Comment (i-vyshnevska) [reply component_event/tests/test_event.py:25]: the field is defined in odoo metatest class
 https://github.com/odoo/odoo/blob/5cfce06bf92cc746b4dc3b2b554ad57daac1af1d/odoo/tests/common.py#L296
 but us we inherit here form uittetst we also need this field so odoo will be able to run test

**Review by gurneyalex — APPROVED** [review]

File: component_event/tests/test_event.py (hunk 5 above)
Comment (yankinmax) [reply component_event/tests/test_event.py:25]: @i-vyshnevska thanks for the explanation

**Review by simahawk — APPROVED** [review]

**Review by simahawk — CHANGES_REQUESTED** [review]
@i-vyshnevska there's a mistake. 
There's a commit to migrate "test_connector" but I don't see "test_component".
Also that commit contains unrelated changes https://github.com/OCA/connector/pull/425/commits/1e0c578cd1bbc2cd50ed8225518deaa046710010
We need test_component here. Plus, I wonder how tests for test_connector

**Review by simahawk — APPROVED** [review]
Ah, my bad, I got confused by names. Re-checked 15.0, LG.

**Pull request merged**